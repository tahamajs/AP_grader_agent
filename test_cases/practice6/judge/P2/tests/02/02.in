POST signup ? username "low_mist" password "meow"
GET show_budget ?
POST increase_budget ? amount 2500
GET show_budget ?
POST logout ?
POST signup ? username "cool_man" password "1234"
POST increase_budget ? amount 1500
GET show_budget ?
POST logout ?
POST login ? username "cool_man" password "1234"
GET show_budget ?
POST increase_budget ? amount 200
GET show_budget ?
POST logout ?
POST login ? username "low_mist" password "meow"
GET show_budget ?
POST increase_budget ? amount 300
GET show_budget ?
POST increase_budget ? amount 400
GET show_budget ?
POST increase_budget ? amount -500
GET show_budget ?
POST logout ?