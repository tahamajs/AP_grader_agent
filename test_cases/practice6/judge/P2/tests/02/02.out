OK
0
OK
2500
OK
OK
OK
1500
OK
OK
1500
OK
1700
OK
OK
2500
OK
2800
OK
3200
Bad Request
3200
OK